Packs/FeedTAXII/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Fixed an issue in **TAXII2ApiModule** where **extension-definition** objects arrived to the system instead of being skipped.
```

Comment:
```suggestion
- Fixed an issue where **extension-definition** objects arrived were fetched as indicators of the wrong type instead of being skipped.
```